apps/docs/src/app/core/component-docs/inline-help/examples/inline-help-example.component.html

```diff
@@ -1,26 +1,26 @@
 <div class="fd-inline-help-example">
     <span>Default</span>
-    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" tabindex="0" ariaLabel="Inline Help Tooltip"></fd-icon>
+    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" tabindex="0" ariaLabel="Default Inline Help Tooltip"></fd-icon>
 </div>
 
 <div class="fd-inline-help-example">
     <span>Left Position</span>
-    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="left" tabindex="0" ariaLabel="Inline Help Tooltip"></fd-icon>
+    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="left" tabindex="0" ariaLabel="Left Position Inline Help Tooltip"></fd-icon>
 </div>
 
 <div class="fd-inline-help-example">
     <span>Right Position</span>
-    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="right" tabindex="0" ariaLabel="Inline Help Tooltip" ></fd-icon>
+    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="right" tabindex="0" ariaLabel="Right Position Inline Help Tooltip" ></fd-icon>
 </div>
 
 <div class="fd-inline-help-example">
     <span>Top Position</span>
-    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="top" tabindex="0" ariaLabel="Inline Help Tooltip"></fd-icon>
+    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="top" tabindex="0" ariaLabel="Top Position Inline Help Tooltip"></fd-icon>
 </div>
 
 <div class="fd-inline-help-example">
     <span>Bottom Position</span>
-    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="bottom" tabindex="0" ariaLabel="Inline Help Tooltip"></fd-icon>
+    <fd-icon glyph="question-mark" fd-inline-help="Inline Help Tooltip" placement="bottom" tabindex="0" ariaLabel="Bottom Position Inline Help Tooltip"></fd-icon>
```

Comment:
Just wondering how helpful is the information "Bottom Position/right position, etc" to a blind person. Like in the other cases, can we set the aria-label value automatically, behind the scene, to the value provided for `fd-inline-help`. If the user wants something different they can set manually the `araiLabel` attribute. @mikerodonnell89 what do you think?

Reply:
@InnaAtanasova ariaLabel is obtained from the icon component, We need to create a new @input variable as InlineAriaLabel cause the ariaLabel is already used in the Fd-icon component.

Reply:
How is ariaLabel "obtained" when you provide a string input?

Reply:
@Lokanathan-k please process this one.